### ISSUE TITLE
Incompatible with psutil 3.0.0

### ISSUE THREAD

**Issue by csdev** [fabianp/memory_profiler#94]
psutil 3.0.0 was recently released. Attempting to profile code using this version of psutil results in an exception:

```
Traceback (most recent call last):
  File "/usr/local/bin/mprof", line 472, in <module>
    actions[get_action()]()
  File "/usr/local/bin/mprof", line 220, in run_action
    include_children=options.include_children, stream=f)
  File "/usr/local/lib/python2.7/dist-packages/memory_profiler.py", line 243, in memory_usage
    include_children=include_children)
  File "/usr/local/lib/python2.7/dist-packages/memory_profiler.py", line 48, in _get_memory
    mem_info = getattr(process, 'memory_info', process.get_memory_info)
AttributeError: 'Process' object has no attribute 'get_memory_info'
```

(memory_profiler does appear to work correctly with the previous version of psutil, 2.2.1)

**Comment by csdev** [comment]
looks like this is related to #79

**Comment by fabianp** [comment]
Hi,

Thanks for reporting this. I released a new version that should fix this. Can you update your memory_profiler to the just released 0.33 (pip install -U memory_profiler) and report whether this solves the problem?

Best,

Fabian

**Comment by csdev** [comment]
Thanks, that update solved the issue.
